node/src/node_configurator/node_configurator_standard.rs

```diff
@@ -707,16 +717,15 @@ mod tests {
     #[test]
     fn server_initializer_collected_params_senses_when_user_specifies_config_file() {
         running_test();
-        let data_dir = ensure_node_home_directory_exists(
-            "node_configurator_standard",
-            "server_initializer_collected_params_senses_when_user_specifies_config_file",
-        );
-        let args = ArgsBuilder::new().param("--config-file", "booga.toml"); // nonexistent config file: should stimulate panic because user-specified
+        let home_dir = PathBuf::from("/unexisting_home/unexisting_alice");
+        let chain_specific_data_dir = add_chain_specific_directories(Chain::PolyMainnet, &home_dir);
+        let args = ArgsBuilder::new()
+            .param("--config-file", "booga.toml") // nonexistent config file: should stimulate panic because user-specified
```

Comment:
Not your fault but please change the wording `should stimulate panic` to `should return error`. The reason is that this whole test doesn't experience any panic at all so I don't like the mention in the comment. :wink: